frontend/src/routes/(authenticated)/admin/+page.ts

```diff
@@ -3,43 +3,70 @@ import { getClient, graphql } from '$lib/gql';
 import type { PageLoadEvent } from './$types';
 import { isAdmin, type LexAuthUser } from '$lib/user';
 import { redirect } from '@sveltejs/kit';
-import { getBoolSearchParam } from '$lib/util/query-params';
-import type { $OpResult, ChangeUserAccountByAdminInput, ChangeUserAccountByAdminMutation, ProjectType } from '$lib/gql/types';
-import type {LoadAdminDashboardQuery} from '$lib/gql/types';
+import { getBoolSearchParam, getSearchParam } from '$lib/util/query-params';
+import type { $OpResult, ChangeUserAccountByAdminInput, ChangeUserAccountByAdminMutation, ProjectFilterInput, ProjectType } from '$lib/gql/types';
+import type { LoadAdminDashboardProjectsQuery, LoadAdminDashboardUsersQuery } from '$lib/gql/types';
 
-export type AdminSearchParams = {
+export const _FILTER_PAGE_SIZE = 100;
+
+export type AdminProjectSearchParams = {
   showDeletedProjects: boolean,
   projectType: ProjectType | undefined,
   userEmail: string | undefined,
   projectSearch: string,
 };
 
-export type Project = LoadAdminDashboardQuery['projects'][number];
-export type User = LoadAdminDashboardQuery['users']['items'][number];
+export type AdminUserSearchParams = {
+  userSearch: string,
+  userEmail: string | undefined,
```

Comment:
As discussed, we will try to use the convention of loading the query params in +page.svelte and then pass them down (potentially in some cases using the context API), because (1) weird things could happen if a query param is initialized twice with different defaults and (2) it sort of makes it more obvious what to call things when multiple components reference the same query-param (wow that sounded convincing 😆)